packages/react-hooks/src/useMountSync.ts

```diff
@@ -12,6 +12,7 @@ import * as React from 'react';
 export const useMountSync = (callback: () => void) => {
```

Comment:
Can you also mark this as `@deprecated`? It's not used anywhere and is fundamentally problematic due to useLayoutEffect usage.

Reply:
Sure, marked it by JSDoc & in README.md 👍